README.md

```diff
@@ -83,7 +83,7 @@ Replace {BRANCH} or {TAG NAME} or \<branch\> with the branch and tag you'd like
 When using a tag-based build, you're essentially cloning a point in time to build that specific version of Crafter CMS. That implies that you won't be able to update/nor push changes back.
 
 # 3. Build a Developer's Environment
-Crafter CMS is built along a microservices architecture, and as such, comprises a number of head-less, RESTful, modules that work together to provide the final solution. In this section, we'll start with the simple case of _build everything_/_run everything_, and then move on to building/hacking individual modules.
+Crafter CMS is built along a microservices based, serverless (Docker + Kubernetes), architecture, and as such, comprises a number of headless API-first (GraphQL, REST, in-process)  modules that work together to provide the final solution. In this section, we'll start with the simple case of _build everything_/_run everything_, and then move on to building/hacking individual modules.
```

Comment:
microservices-based

Reply:
We must also indicate that you don't have to run Kuber to get a local env.

A basic dev env, like this section, is probably not going to be based on a Kuber cluster. Let's discuss.

Reply:
Yeah I don't think we need to reference Kuber or Docker hear except to say that you can build the Docker images.